src/components/Create/components/pages/ReviewDeploy/components/ProjectDetailsReview/ProjectDetailsReview.tsx

```diff
@@ -83,6 +84,17 @@ export const ProjectDetailsReview = () => {
           ) : null
         }
       />
+      {projectRequiredOFACCheck && (
+        <ReviewDescription
+          className="col-span-3"
+          title={t`OFAC Sanctions screening`}
+          desc={
+            <div className="overflow-hidden text-ellipsis text-base font-medium">
+              {String(projectRequiredOFACCheck)}
```

Comment:
If undefined, this will show as `"undefined"`. Better to use explicit change, or set a default value somewhere

Reply:
if there is `undefiend`, the component will not showing (look at 87 row)

I thought I'd check it out
<img width="763" alt="image" src="https://github.com/jbx-protocol/juice-interface/assets/147380121/1dd454c8-d992-4826-a206-481f3dc696c1">

there is nothing to display

<img width="970" alt="image" src="https://github.com/jbx-protocol/juice-interface/assets/147380121/188b4125-d37e-47cc-bcb4-fdb10509a419">

If the variable is a bool, it looks like this
<img width="946" alt="image" src="https://github.com/jbx-protocol/juice-interface/assets/147380121/36092bda-deab-4b3f-b0c9-ac4fbb8770ee">

Reply:
If you have internal rules/code style for such cases, please describe it to me, I will fix it according to your rules/code style.